docs/site/src/demos/app-bar/app-bar.md

```diff
@@ -0,0 +1,11 @@
+# App Bar
+
+The [app-bar](https://material.io/guidelines/layout/structure.html#structure-app-bar), formerly known as the action bar in Android, is a special kind of toolbar that’s used for branding, navigation, search, and actions.
```

Comment:
I was going to say something similar, except that they're "Bottom navigation", "Selection controls" etc. which follows the MD guideline site.